xcdat/regridder/regrid2.py

```diff
@@ -172,7 +172,12 @@ def _output_axis_sizes(self, da: xr.DataArray) -> Dict[str, int]:
         axis_name_map = {y[0]: x for x, y in da.cf.axes.items()}
 
         for standard_name in da.sizes.keys():
-            axis_name = axis_name_map[standard_name]
+            try:
+                axis_name = axis_name_map[standard_name]
+            except KeyError:
+                raise RuntimeError(
+                    f"Could not find axis {standard_name!r}, ensure {standard_name!r} exists and the attributes are correct."
+                )
```

Comment:
```suggestion
                raise RuntimeError(
                    f"Could not find axis {standard_name!r}, ensure {standard_name!r} "
                    "exists and the attributes are correct."
                )
```

`black` doesn't reformat string line length > limit (configured to 88) [yet](https://github.com/psf/black/issues/1802).